src/navigators/MainTabNavigator.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Icon from '@expo/vector-icons/FontAwesome';
```

Comment:
react-native-vector-icons has nice sets too if you wanna use

Reply:
ya this is a wrapper around that for expo!